spec/fixtures/maxretry_worker.rb

```diff
@@ -0,0 +1,45 @@
+require 'sneakers'
+require 'thread'
+require 'redis'
+
+require 'sneakers/handlers/maxretry'
+
+# This worker ... works
```

Comment:
I think I would call these workers something more along the lines of how they behave: `AlwaysAckWorker`, `AlwaysRejectWorker`, `RejectOnceWorker` (which is what you have).  It may make them more portable across other tests.